alvr/dashboard/src/dashboard/components/settings_controls/presets/builtin_schema.rs

```diff
@@ -32,7 +32,12 @@ fn bool_modifier(target_path: &str, value: bool) -> PresetModifier {
 pub fn resolution_schema() -> PresetSchemaNode {
     PresetSchemaNode::HigherOrderChoice(HigherOrderChoiceSchema {
         name: "resolution".into(),
-        strings: HashMap::new(),
+        strings: [(
+            "help".into(),
+            "Choosing too high resolution may result in high latency or black screen.".into(),
```

Comment:
Suggestion:
> Choosing too high of a resolution (commonly 4096x4096)...
> Choosing a resolution that's too high (i.e. 4096x4096) ...

Reply:
not all headsets are square though, i think sticking to presets (for ex. `width: 5184)`) is better

Reply:
Yes but that's the resolution where stuff breaks, but yea idk, it's annoying how this interacts with foveated encoding, this should be checked in the actual code implementation not here. You can still do the grammar improvements tho

Reply:
yeah... just thought to add at least some basic food for thought to users without changing code significantly

Ideally dashboard (or client) should tell if encoder or decoder are overloaded or simply don't work with such resolutions
I was thinking of doing some basic checks at runtime that will make a note (similar to SteamVR not running) in statistics when user has too high decode/encode, network, game render, etc latency, maybe should move that into separate issue so i won't forget for later